### PR TITLE
Add aria labels to '•••' buttons in dacfx and schema compare

### PR DESCRIPTION
**Pull request by kisantia**
Adding aria labels to the '•••' buttons so that their names show as "Select file". Addresses #6742.

image

image

### CODE REVIEW

File: extensions/schema-compare/src/dialogs/schemaCompareDialog.ts [hunk 1]
```diff
@@ -288,10 +288,12 @@ export class SchemaCompareDialog {
 		if (isTarget) {
 			this.targetFileButton = view.modelBuilder.button().withProperties({
 				label: '•••',
+				ariaLabel: localize('schemaCompare.selectFileAriaLabel', "Select file")
```
Comment (Benjin) [inline extensions/schema-compare/src/dialogs/schemaCompareDialog.ts]: These should probably be more descriptive, like "Select target file" and "Select source file" so someone can differentiate between them.

File: extensions/schema-compare/src/dialogs/schemaCompareDialog.ts [hunk 2]
```diff
@@ -288,10 +288,12 @@ export class SchemaCompareDialog {
 		if (isTarget) {
 			this.targetFileButton = view.modelBuilder.button().withProperties({
 				label: '•••',
```
Comment (Benjin) [inline extensions/schema-compare/src/dialogs/schemaCompareDialog.ts]: Probs a good opportunity to add a tooltip, too, with the same text as the ariaLabel

File: extensions/schema-compare/src/dialogs/schemaCompareDialog.ts (hunk 1 above)
Comment (Benjin) [inline extensions/schema-compare/src/dialogs/schemaCompareDialog.ts]: These should probably be more descriptive, like "Select target file" and "Select source file" so someone can differentiate between them.

File: extensions/schema-compare/src/dialogs/schemaCompareDialog.ts [hunk 3]
```diff
@@ -288,12 +288,14 @@ export class SchemaCompareDialog {
...
-				ariaLabel: localize('schemaCompare.selectFileAriaLabel', "Select file")
+				title: localize('schemaCompare.selectTargetFile', "Select target file"),
+				ariaLabel: localize('schemaCompare.selectTargetFile', "Select target file")
 			}).component();
 		} else {
 			this.sourceFileButton = view.modelBuilder.button().withProperties({
 				label: '•••',
-				ariaLabel: localize('schemaCompare.selectFileAriaLabel', "Select file")
+				title: localize('schemaCompare.selectTargetFile', "Select target file"),
```
Comment (Benjin) [inline extensions/schema-compare/src/dialogs/schemaCompareDialog.ts]: 🚨 Copy-pasta!  Should be "source file"

File: extensions/dacpac/src/wizard/api/dacFxConfigPage.ts [hunk 4]
```diff
@@ -162,6 +162,8 @@ export abstract class DacFxConfigPage extends BasePage {
 		this.fileTextBox.ariaLabel = localize('dacfx.fileLocationAriaLabel', "File Location");
 		this.fileButton = this.view.modelBuilder.button().withProperties({
 			label: '•••',
```
Comment (alanrenmsft) [inline extensions/dacpac/src/wizard/api/dacFxConfigPage.ts]: should we use … (ellipsis) or 3 dots?

Comment (kisantia) [reply extensions/dacpac/src/wizard/api/dacFxConfigPage.ts]: this is what Smitha told us use when we first made this wizard.

**Pull request merged**